src/set_theory/cardinal_ordinal.lean

```diff
@@ -229,6 +230,50 @@ by rw [← aleph_zero, ← aleph_succ, ordinal.succ_zero]
 lemma countable_iff_lt_aleph_one {α : Type*} (s : set α) : countable s ↔ #s < aleph 1 :=
 by rw [← succ_omega, lt_succ, mk_set_le_omega]
 
+private lemma ord_mem {c : cardinal} : c.ord.card.ord = c.ord :=
+by rw card_ord
```

Comment:
Can this lemma be inlined into its two occurrences?

Reply:
That doesn't work directly, since where it's used, the actual goal is too messy for the rewrite to work. I can inline it as `(by { dsimp, rw card_ord })`, but I'd personally prefer taking the hit with the private lemma.

Reply:
Eh, I decided to actually inline it.